work_queue/src/work_queue_graph_log

```diff
@@ -75,29 +75,12 @@ def read_log_entries(file, fields):
 
             log_entries[record['timestamp']] = record
 
-            record['time_master']   = record['time_status_msgs'] + record['time_internal'] + record['time_application'] + record['time_send'] + record['time_receive']
-            record['bytes_sent'] /= unit_factors['GB']
-            record['bytes_received'] /= unit_factors['GB']
-            record['bytes_transfered'] = record['bytes_sent'] + record['bytes_received']
-
-            try:
-                record['capacity_tasks'] = (record['time_execute'] + record['time_polling']) / record['time_master']
-            except ArithmeticError:
-                record['capacity_tasks'] = 0
-
+            record['time_master']   = record['time_send'] + record['time_receive']
```

Comment:
This time_master is different from the one computed in work_queue. Either make them the same, or give more appropriate names.